cylc/flow/task_pool.py

```diff
@@ -395,7 +400,7 @@ def compute_runahead(self, force=False) -> bool:
         if self.stop_point and limit_point > self.stop_point:
             limit_point = self.stop_point
             LOG.debug(f"{pre_adj_limit} -> {limit_point} (stop point)")
-        LOG.info(f"Runahead limit: {limit_point}")
```

Comment:
Can we just remove this one?

The processed runahead limit can be found in the processed configuration file. I don't think this message is adding anything?

Reply:
Correct, that logs the new concrete runahead limit when it changes. I suppose that could go to DEBUG though.